frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -106,14 +107,15 @@ local footerTextGeneratorMap = {
         local symbol_type = footer.settings.item_prefix or "icons"
         local prefix = symbol_prefix[symbol_type].frontlight
         local powerd = Device:getPowerDevice()
-        if powerd:isFrontlightOn() then
+        if powerd:isFrontlightOn() and powerd:getFrontlightSwitchState() then
```

Comment:
I would like to include getFrontlightSwitchState as part of the return value of isFrontlightOn and leave this as is

Reply:
Good point.